### PR TITLE
Remove reader/start from state tree

### PR DESCRIPTION
**Pull request by bluefuton**
After removing the Reader Cold Start introductory experience from Calypso, we kept some of the Cold Start Redux code to potentially re-use with the following intro banner (https://github.com/Automattic/wp-calypso/pull/11282).

It now looks like we will use Calypso preferences for that instead (see https://github.com/Automattic/wp-calypso/pull/13184#issue-222451233), so this PR removes the remaining Redux code for Cold Start.

### CODE REVIEW

File: client/reader/following/intro.jsx [hunk 1]
```diff
@@ -3,22 +3,20 @@
  */
 import React from 'react';
 import { localize } from 'i18n-calypso';
-import { connect } from 'react-redux';
```
Comment (samouri) [inline client/reader/following/intro.jsx]: why are you commenting instead of removing?

Comment (bluefuton) [inline client/reader/following/intro.jsx]: We're going to `connect` it to something else very soon (`state/preferences`).

Comment (samouri) [inline client/reader/following/intro.jsx]: will we still be using `props.requestGraduate` etc.?
I'd prefer we removed those lines even if adding them back in shortly

Comment (bluefuton) [inline client/reader/following/intro.jsx]: No - we're going to use Calypso preferences instead, probably adding a new action called `deletePreference` which performs the same function as `requestGraduate`.

I'm happy to remove the comment lines in the meantime though 👍

Comment (samouri) [inline client/reader/following/intro.jsx]: Thanks @bluefuton, i like todos. but get scared by commented out code

**Comment by bluefuton** [comment]
@samouri good for :ship:?

**Comment by samouri** [comment]
@bluefuton, i'll give it a quick test run now

**Comment by samouri** [comment]
🚢

**Pull request merged**